### DISCUSSION TITLE
jpackage and wixtoolset

### DISCUSSION THREAD

**Discussion by jeromebridonneau** [wixtoolset/issues#6384] [Questions]
Hello,

Below is the message I have since yesterday when I use jpackage (JDK14), 
3 weeks ago jpackage was working perfectly and I didn't do anything except an automatic windows update, 
now I can only use the "jpackage --type app-image" command. Is this problem due to updating windows (I can't go back to the old one unfortunately?

I have WiX Toolset v3.11.2.4516 with Windows 10 Famille version  O S 19041.804

I use jpackage (JDK14) and the following commands:
jpackage --type msi --copyright "©Jérôme Bridonneau 2005-2021" --icon "input\logo.ico" --win-shortcut --win-console --win-dir-chooser 
--app-version "6" --vendor "Jérôme Bridonneau" --dest output --name PModularSynthesizer6 --input input --main-jar PModularSynthesizerV6_protect.jar

WARNING: Using incubator modules: jdk.incubator.jpackage

The message I have :

java.io.IOException: Command [C:\Program Files (x86)\WiX Toolset v3.11\bin\candle.exe, -nologo, 
C:\Users\JRME~1\AppData\Local\Temp\jdk.incubator.jpackage6196540313233711278\config\main.wxs, -ext, WixUtilExtension, -arch, x64, -out, 
C:\Users\JRME~1\AppData\Local\Temp\jdk.incubator.jpackage6196540313233711278\wixobj\main.wixobj, 
-dJpAppDescription=PModularSynthesizer6, 
-dJpAppVersion=6, 
-dJpInstallDirChooser=yes, 
-dJpProductCode=6dfc863e-7c84-3d10-b125-2134994c1933, 
-dJpAppName=PModularSynthesizer6, 
-dJpAllowUpgrades=yes, 
-dJpProductUpgradeCode=e2c0347c-1f82-339d-9687-93562512e97c, 
-dJpIsSystemWide=yes, 
-dJpAppVendor=Jérôme Bridonneau, 
-dJpConfigDir=C:\Users\JRME~1\AppData\Local\Temp\jdk.incubator.jpackage6196540313233711278\config]
in C:\Users\JRME~1\AppData\Local\Temp\jdk.incubator.jpackage6196540313233711278\images\win-msi.image\PModularSynthesizer6 
exited with 10 code

the problem apparently comes from wix toolset

Thank you for your help

Jérôme

**Comment by barnson** [comment]
WiX doesn't use issues for support. We moved this issue to the [experimental GitHub discussions area][gh].

[gh]: https://github.com/wixtoolset/issues/discussions?discussions_q=category%3AQuestions
